### PR TITLE
An example specification of the Cosmos SDK bank module

### PR DESCRIPTION
**Pull request by konnov**
I've prototyped this specification in January. Since then we have added plenty of good features in the language. It feels to be the right time for merging this spec.

The distinct feature of this example is literate programming. Both the functional specification and the state machine are introduced in the markdown text directly. So the markdown file is the only file that actually needs a review. The text description of the bank module is copied from Cosmos SDK. We do not have to proof-read text issues, if there are any.

This specification could be further improved. For instance, we could more tests. However, it is already large enough. It could still be interesting to the people who are looking at the bank module.

### CODE REVIEW

File: examples/cosmos/bank/bankTest.qnt [hunk 1]
```diff
@@ -96,12 +98,13 @@ module bankTest {
...
     }
   }
+
   run sendTest = {
     init
       // the King has bananas and he can send it
       .then(send("king",
                  "donkeykong",
-                 Set("banana").mapBy(d => 2_000)))
+                 Map("banana" -> 2_000)))
```
Comment (konnov) [inline examples/cosmos/bank/bankTest.qnt:107]: Thanks! This indeed looks much simpler :)

Comment (bugarela) [reply examples/cosmos/bank/bankTest.qnt:107]: I just realized that I need to update the markdown, and then re-generate the file. I'm doing that right now!

**Review by bugarela — APPROVED** [review]
I didn't look into much detail, but it looks good to me!

**Comment by konnov** [comment]
Thanks! Then let's merge it. I have enabled automerge.

**Comment by bugarela** [comment]
For some reason, auto-merge didn't work, so I merged it manually

**Pull request merged**